server/server.ts

```diff
@@ -77,10 +78,6 @@ async function init() {
     saveRoomsToRedis();
   }
 
```

Comment:
Actually if you don't mind I'd like to keep this default room. I have another project that currently depends on that existing and is based on this code.